lib/hotwire_combobox/helper.rb

```diff
@@ -122,7 +122,30 @@ def hw_call_method_or_proc(object, method_or_proc)
         if method_or_proc.is_a? Proc
           method_or_proc.call object
         else
-          object.public_send method_or_proc
+          hw_call_method object, method_or_proc
+        end
+      end
+
+      def hw_call_method(object, method)
+        if object.respond_to? method
+          object.public_send method
+        else
+          raise NoMethodError, <<~MSG
+            Message from HotwireCombobox: [ACTION NEEDED]
+
+            `#{object.class}` does not respond to `##{method}`.
```

Comment:
Actually, with the current implementation, private methods would land here, too. In that case, it's not that the class doesn't have a method. But, rather, that the method is private.

Unsure if we should just let you call private methods here, of if we should also catch this kind of error and return a similar friendly message.

Reply:
I'm thinking let's just let people call private methods here by using `send` instead of `public_send`. The library shouldn't force you to make a method public if you really don't want to.

Reply:
You could probably mention it needs to be a public method in the message to account for that somewhat. I don't feel like you'd want to call private methods on the model for display generally. If you did, you could call it from the public `to_combobox_display` method anyways.

Reply:
That's a good point about calling it from `#to_combobox_display`. Sounds good!